Automattic/newspack-theme
# Update node-sass and calypso-build to support node v.12

### All Submissions:






### Changes proposed in this Pull Request:



Upgrade `node-sass` to `4.13.0` and `@automattic/calypso-build` to `5.1.0` in order to make it possible to compile on node version `12`

### How to test the changes in this Pull Request:

1. Set node version to 12 (`nvm use 12`)
2. Install node dependencies (`npm i`)
3. Compile code (`npm run build`)
4. All should work without issues. You _might_ have to remove `node_modules` first

### Other information:
